independent-publisher-connectors/FreeAgent/apiDefinition.swagger.json

```diff
@@ -0,0 +1,1846 @@
+            "name": "contact_id",
+            "default": "",
+            "in": "path",
+            "x-ms-url-encoding": "single",
+            "type": "string",
+            "x-ms-summary": "Contact",
+            "description": "Contact",
+            "required": true,
+            "x-ms-dynamic-values": {
+              "operationId": "GetContacts",
+              "value-collection": "contacts",
+              "value-path": "id",
+              "value-title": "organisation_name"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "200",
+            "schema": {}
+          }
+        }
+      },
+      "put": {
+        "summary": "Update Contact",
+        "x-ms-summary": "Update Contact",
+        "description": "Update Contact",
+        "operationId": "UpdateContact",
+        "consumes": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "name": "contact_id",
+            "default": "",
+            "in": "path",
+            "x-ms-url-encoding": "single",
+            "type": "string",
+            "x-ms-summary": "Contact",
+            "description": "Contact",
+            "x-ms-dynamic-values": {
+              "operationId": "GetContacts",
+              "value-collection": "contacts",
+              "value-path": "id",
+              "value-title": "organisation_name"
+            },
+            "required": true
+          },
+          {
+            "name": "body",
+            "in": "body",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "contact": {
+                  "type": "object",
+                  "properties": {
+                    "contact_name_on_invoices": {
+                      "type": "boolean",
+                      "x-ms-summary": "Contact Name On Invoices",
+                      "description": "Contact Name On Invoices"
+                    },
+                    "default_payment_terms_in_days": {
+                      "type": "integer",
+                      "format": "int32",
+                      "x-ms-summary": "Default Payment Terms In Days",
+                      "description": "Default Payment Terms In Days"
+                    },
+                    "locale": {
+                      "type": "string",
+                      "x-ms-summary": "Locale",
+                      "description": "Locale"
+                    },
+                    "country": {
+                      "type": "string",
+                      "x-ms-summary": "Country",
+                      "description": "Country"
+                    }
+                  },
+                  "x-ms-summary": "Contact",
+                  "description": "Contact"
+                }
+              }
+            },
+            "required": true
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "200",
+            "schema": {}
```

Comment:
@nk-gears It came lately into the focus. Empty response schema are not allowed any more, which means no dynamic content will be shown in the output and makers have to use parse JSON to parse the response. Please provide valid response schema definition for all the operations in the swagger.

Reply:
To add to what Srikanth mentioned, if there is a reason why you added a response schema, please let us know.

Reply:
@sriyen-msft @natalie-pienkowska  Thanks for the details. I didn't know that empty schema are not allowed. The underlying API doesn't returns any data for some of the operations like delete ,put and patch.   

For these type of operations shall i just use like the below?. (i.e) using a "default" response without a schema property. (instead of having empty schema with the property)

```

  "responses": {
          "default": {
            "description": "default"
          }
        }
          
```

Reply:
@nk-gears Thanks for providing schema definitions for most of the responses in the swagger. I believe it is ok to have single response code for delete, put and similar responses. But I will let you know if our engineering team push back in the certification process.